oci_python_image/hello_world/test.yaml

```diff
@@ -0,0 +1,6 @@
+# See https://github.com/GoogleContainerTools/container-structure-test#command-tests
+schemaVersion: 2.0.0
+commandTests:
+  - name: 'run'
+    command: /opt/hello_world/hello_world_bin
```

Comment:
Can you omit this? We want the test to assert is the default entrypoint is correct.

Reply:
No, you can't:
https://github.com/GoogleContainerTools/container-structure-test#command-tests

but, we can add a metadataTest which checks the entrypoint, will add in a follow-up

Reply:
Interesting. Thanks for pointing it out.